.github/workflows/ci.yml

```diff
@@ -25,6 +25,8 @@ jobs:
       - uses: dtolnay/rust-toolchain@stable
       - run: cargo build
         name: Build
+      - run: cargo build --features variable_fonts
```

Comment:
variable_fonts is already the default feature, so we should rather test with no default features above, right?